generators/Exercises/Hamming.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Generators.Input;
+
+namespace Generators.Exercises
+{
+    public class Hamming : Exercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            foreach (var canonicalDataCase in CanonicalData.Cases)
+            {
+                canonicalDataCase.ExceptionThrown = canonicalDataCase.Expected is long ? null : typeof(ArgumentException);
+            }
```

Comment:
Ah I see yes :) We have to deserialize it as a `long`, but afterwards we could convert the long to an int if having an int was more convenient for us with `Convert.Int32`.

It might be helpful to due in future exercises if there are more complex operations that we need to do with that deserialized value.